### DISCUSSION TITLE
Welcome to 007-TheBond Discussions!

### DISCUSSION THREAD

**Discussion by Deadshot0x7** [Deadshot0x7/007-TheBond#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Elina235** [comment]
Hello, I found the issue. In Termux after searching something is close all founded information instantly

**Reply by Deadshot0x7** [reply]
Thanks for bringing attention to the issue. The issue  has been fixed  will push in the next release, Till the enjoy with others at discord community https://discord.gg/ECyeQK5s

**Reply by tgeniusclub** [reply]
Hello dear @Deadshot0x7 and @kana800 I really appreciate your effort in making this amazing tool, I'm also a termux user and a blogger. I wrote a blog post about your tool [007-Thebond](https://tgeniusclub.com/gather-information-about-your-victim/) and I will be happy if you add my link to your README. Thanks and may God help you and gives you the strength to make other good tools. kindly check https://tgeniusclub.com/gather-information-about-your-victim/

**Reply by Deadshot0x7** [reply]
Hey @tgeniusclub  , Thanks for the covering the tool on your blog.

**Reply by kittycatrawr** [reply]
i did everything, but when im ready to open it says no file or directory

**Comment by tgeniusclub** [comment]
you are welcome and thanks for you your quick reply. don't hesitate to let
me know if you are any other tool so that I can write about it

Le mer. 23 nov. 2022 à 19:33, DeadShot0x7 ***@***.***> a
écrit :

**Comment by dedswagger** [comment]
pip install -r requirements.txt
ERROR: Could not find a version that satisfies the requirement email-validate (from versions: none)
ERROR: No matching distribution found for email-validate

**Comment by Milkshake2022** [comment]
Traceback (most recent call last):
File "/home/kali/007-TheBond/scripts/007-TheBond.py", line 2, in
import pyfiglet
ModuleNotFoundError: No module named 'pyfiglet'

I hope you can help me

thank you
